ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -91,8 +91,7 @@ public FastSyncDownloader(
     if (ExceptionUtils.rootCause(error) instanceof FastSyncException) {
       return CompletableFuture.failedFuture(error);
     } else if (ExceptionUtils.rootCause(error) instanceof StalledDownloadException) {
-      LOG.warn(
-          "Fast sync was unable to download the world state. Retrying with a new pivot block.");
+      LOG.info("Re-pivoting to newer block.");
```

Comment:
We fail if we repivot 25 times so I don't think we need to have a warning that sounds so dire here when it happens naturally over the course of a fast-sync.